code/game/gamemodes/clock_cult/clock_effects/spatial_gateway.dm

```diff
@@ -119,7 +119,7 @@
 		qdel(src)
 		return FALSE
 	if(!sender)
-		visible_message("<span class='warning'>[A] bounces off of [src]!</span>")
+		visible_message("<span class='warning'>[A] bounces off [src]!</span>")
```

Comment:
"bounces off of the x" sounds better than" bounces off the x", though, even if it is incorrect, imo.